### PR TITLE
Branches

### PR DESCRIPTION
**Pull request by SSSpac**
I added some info about branches + some pictures. 
Let me know what needs change if there is :)

### CODE REVIEW

File: branches.html [hunk 1]
```diff
@@ -8,15 +8,17 @@
...
+            <img class="second__image" src="./pictures/img2.png" alt="info" height="105%" width="100%">              
     </main>
-
-    <footer class="footer">
-
-    </footer>
+<footer class="footer">
+        <p> this is the footer</p>
+</footer>
```
Comment (Alexander-Lindstroem) [inline branches.html:22]: The footer should still be indented

Comment (PriyeshIllam) [inline branches.html:22]: LGTM

**Review by PriyeshIllam — APPROVED** [review]
LGTM

**Review by Maatala — APPROVED** [review]
LGTM

**Review by idacarlin95 — APPROVED** [review]
LGTM!

**Pull request merged**